code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -258,3 +279,102 @@
 	new /obj/item/weapon/spacecash/c1000(src)
 	new /obj/item/weapon/spacecash/c1000(src)
 	new /obj/item/weapon/spacecash/c1000(src)
+
+//Value Packs
+/obj/item/weapon/storage/backpack/clown/tratior/fill()
+	..()
+	new /obj/item/clothing/mask/gas/voice/clown(src)
+	new /obj/item/clothing/under/rank/clown/traitor(src) //About 2
+	new /obj/item/clothing/shoes/clown_shoes/traitor(src) //About 2
+	new /obj/item/clothing/gloves/force/clown(src) //About 6
+	new /obj/item/device/firing_pin/clown(src) //Free because why
```

Comment:
This won't even work without the user being a clown job-wise.

Reply:
I've seen some clown gimmicks work. Before you purchase this, you can ask to become a clown via HoP.

Or alternatively, a nameless, jobless clown that hides in maint.

Reply:
Actually, no. It's impossible to fire any gun with this pin. Literally all it's useful is for rendering guns unusable.

Reply:
Oh, I thought you were referring to the gimmick existing itself, not the firing pin.